docs/installation/ubuntulinux.md

```diff
@@ -349,6 +349,12 @@ To avoid this warning, you can specify a DNS server for use by Docker
 containers. Or, you can disable `dnsmasq` in NetworkManager. Though, disabling
 `dnsmasq` might make DNS resolution slower on some networks.
 
+
+The instructions below describe how to configure a daemon running on
+Ubuntu 14.10 or below. Ubuntu 15.04 and above uses systemd as its boot
+and service manager. Refer to [control and configure Docker with systemd](../articles/systemd.md#custom-docker-daemon-options)
+to configure a daemon controled by systemd.
```

Comment:
Looks like you missed some changes (e.g. "a daemon" -> "the Docker daemon"). I also see a typo; controled -> controlled.

Can you copy/paste @moxiegirl's example?